README.md

```diff
@@ -415,19 +415,25 @@ We strive to completely cover all applicable issues with these core labels:
    * __[S-Tolerable](https://github.com/vector-im/element-web/labels/S-Tolerable):__
      Purely cosmetic or low / no impact to users
 
-3. __Priority__ — All issues which are not `T-Other` are assigned a priority:
-   * __[P1](https://github.com/vector-im/element-web/labels/P1):__ Next
-   * __[P2](https://github.com/vector-im/element-web/labels/P2):__ Later
-   * __[P3](https://github.com/vector-im/element-web/labels/P3):__ Eventually
-   * __[P4](https://github.com/vector-im/element-web/labels/P4):__ Interesting —
-     Not yet scheduled, will accept patches
-   * __[P5](https://github.com/vector-im/element-web/labels/P5):__ Dubious —
-     Will not schedule, would consider patches
+3. __Prevalence__ — All issues labeled `T-Defect` are also assigned a prevalence:
+   * __[P-High](https://github.com/vector-im/element-web/labels/P-High):__ Affects most users regularly or impacts most users' first experience (e.g. registration) 
+   * __[P-Medium](https://github.com/vector-im/element-web/labels/P-Medium):__ Affects some users regularly or most users rarely
+   * __[P-Low](https://github.com/vector-im/element-web/labels/P-Low):__ Most users are unlikely to come across this or it is a corner case
+   This label may also be used for other types of issue.
 
 4. __Area__ — Most issues are assigned one or several "areas" using one of the
    many `A-` prefixed labels, e.g. `A-Composer` or `A-Spaces`. Each area label
    maps to a group of features or portion of the UI surface in the app.
 
+The Priority label is now deprecated. Defects will now be classed based on their severity and prevalence:
+| Labels    | Equivalent priority | What it means |
+| ----------- | ----------- | ----------- |
+| S‑Critical&nbsp;and&nbsp;F‑High<br />S‑Critical&nbsp;and&nbsp;F‑Medium<br />S‑Major&nbsp;and&nbsp;F‑High | P1        | These issues should be worked on in this sprint or next sprint. If the backlog of issues is too long, we should reevaluate why the bugs are not caught earlier. |
+| S‑Critical and F‑Low<br />S‑Major and F‑Medium<br />S‑Minor and F‑High     | P2        | When all the highest priority bugs are done, this is the next set to tackle. Ideally we should be fixing a few issues from this group every week. |
+| S‑Major and F‑Low<br />S‑Minor and F‑Medium<br />S‑Tolerable and F‑High    | P3        | These issues are wishful thinking. We hope to get to them one day, but we they are low priority. There are likely to be some good new contributor issues in here. |
+| S‑Minor and F‑Low<br />S‑Tolerable&nbsp;and&nbsp;F‑Medium<br />S‑Minor and F‑Low     | P4 and P5 | These issues are unlikely to be actively looked at by the webapp team, but may be picked up by community. |
```

Comment:
Could we also use `P-...` something here?

Reply:
Ah, sorry, I changed what I wanted to call them half way through writing that

Reply:
No worries!